unit-tests/func/serializable-device/test-l500-json-load.py

```diff
@@ -14,12 +14,48 @@
 visual_preset_number = depth_sensor.get_option(rs.option.visual_preset)
 visual_preset_name = rs.l500_visual_preset(int(visual_preset_number))
 
+def json_to_dict( json ):
+    """
+    :param json: a string representing a json file
+    :return: a dictionary with all settings
+    """
+    json_dict = {}
+    for line in json.splitlines():
+        if ':' not in line: # ignoring lines that are not for settings such as empty lines
+            continue
+        setting, value = line.split(':')
+        json_dict[ setting.strip() ] = value.strip()
+    return json_dict
+
+def log_settings_differences( data ):
+    global depth_sensor, sd
+    depth_sensor.set_option(rs.option.visual_preset, int(rs.l500_visual_preset.low_ambient_light))
+    actual_data = str( sd.serialize_json() )
+    data_dict = json_to_dict( data )
+    actual_data_dict = json_to_dict( actual_data )
+    log.debug_indent()
+    try:
+        # logging the differences in the settings between the expected and the actual values
+        log.d( "Printing differences between expected and actual settings values:" )
+        for key in actual_data_dict.keys():
+            if key not in data_dict:
+                log.d( "New setting added to json:", key)
+            elif "Visual Preset" in key or "Temperature" in key or "temperature" in key:
+                # the line regarding the visual preset will always be different because we load 1 from data but set it to
+                # 3 for low ambient. Also all lines regarding temperatures depend on the camera and don't affect the preset
+                continue
+            elif data_dict[ key ] != actual_data_dict[ key ]:
+                log.d( "Setting", key, "was expected to have value of", data_dict[ key ],
+                       "but actually had value of", actual_data_dict[ key ])
+    finally:
+        log.debug_unindent()
+
 #############################################################################################
 # This test checks backward compatibility to old json files that saved with default preset
 # The default preset is deprecated but json files that saved with default preset
-# should be support
+# should be supported
```

Comment:
Add:
```
# Here, we expect the data to fit a pre-defined "low-ambient" preset
# Note that, because FW defines them, the defaults can change from version to version and we need to keep this
# up-to-date with the latest. If the FW changes, this test will fail and output (in debug mode) any settings that
# changed...
```